src/components/application_manager/include/application_manager/policies/policy_handler.h

```diff
@@ -83,7 +83,8 @@ class PolicyHandler : public PolicyHandlerInterface,
   bool ResetPolicyTable() OVERRIDE;
   bool ClearUserConsent() OVERRIDE;
   bool SendMessageToSDK(const BinaryMessage& pt_string,
-                        const std::string& url) OVERRIDE;
+                        const std::string& url,
+                        const uint32_t app_id) OVERRIDE;
```

Comment:
@ZhdanovP What the reason use const modifier if you are not pass parameter through reference

Reply:
@KVGrygoriev The const modifier just prevent some accidental change of this parameter in function.